sequence_processing_pipeline/NuQCJob.py

```diff
@@ -344,6 +350,9 @@ def run(self, callback=None):
                                            empty_files_directory,
                                            self.minimum_bytes)
 
+            # Keep seqs.movi.txt and move it somewhere safe.
```

Comment:
are these needed?

Reply:
I generally like comments but if you feel it's too obvious I can remove it.